### PR TITLE
Add jsx addon

### PR DESCRIPTION
**Pull request by kevinbarabash**
This addon allows developers to see the JSX source code for all of our
stories.  This makes storybook a valuable documentation tool for developers.
    
Test Plan:
    - npm run storybook
    - load http://localhost:9001
    - open the "jsx" tab in the addon panel
    - see the JSX for the story
    
Reviewers: john idr miguel

### CODE REVIEW

**Comment by kevinbarabash** [comment]
screen shot 2018-03-08 at 2 55 34 pm

**Comment by kevinbarabash** [comment]
There were multiple copies of the doc bundles in the docs folder, this PR also cleans that up.

File: .storybook/addons.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import 'storybook-addon-jsx/register';
+import '@storybook/addon-actions/register';
```
Comment (MiguelCastillo) [inline .storybook/addons.js]: i thought this addon was for tracing events from actions?

Comment (kevinbarabash) [reply .storybook/addons.js]: That's correct.  I can add that in a separate PR.

Comment (MiguelCastillo) [reply .storybook/addons.js]: yeah no problem.  I figure we were adding it.  It just made me searching in the PR for events. :)

**Comment by MiguelCastillo** [comment]
this looks good to me!

**Pull request merged**